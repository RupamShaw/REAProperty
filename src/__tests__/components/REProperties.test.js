import React from 'react';
import ReactDOM from 'react-dom';
import ConnectedProperties,{ REProperties }  from '../../components/REProperties';
import { shallow, mount } from 'enzyme';
import { data } from '../../datasource/fixtures.js'

const results = data.results
const saved = data.saved
const props = { results, saved }
const wrapper = shallow(<REProperties {...props} />)

describe('>>>REProperties Initial rendered testing', () => {
    it('REProperties renders correctly', () => {
        expect(wrapper).toMatchSnapshot() //?
    });

    it('Should render successfully', () => {
        expect(wrapper.exists()).toBe(true)
    });

    it('REProperties renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<REProperties {...props} />, div)
        ReactDOM.unmountComponentAtNode(div)
    });
    
    it("if zero  result  i.e count of REProperty tag is zero", (done) => {
        let propszero = {results:[],saved:[]}
        let wrapperPropResult = shallow(<REProperties {...propszero} />)
        // REProperty tag zero"
        expect(wrapperPropResult.find('AdminContact').length).toEqual(1)
        done()
    })

    it("count of REProperty tag is equal to count of data.results+data.saved", (done) => {
        // REProperty tag totalcount is data.results+data.saved"
        expect(wrapper.find('REProperty').length).toEqual(results.length + saved.length)
        //render REProperty tag for resultsa
        expect(wrapper.find('#results REProperty').length).toEqual(results.length)
        //render REProperty tag for saved
        expect(wrapper.find('#saved REProperty').length).toEqual(saved.length)
        done()
    })
})
    