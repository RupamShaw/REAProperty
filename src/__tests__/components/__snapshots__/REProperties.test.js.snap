// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>REProperties Initial rendered testing REProperties renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <REProperties
    results={
        Array [
            Object {
              "agency": Object {
                "brandingColors": Object {
                  "primary": "#ffe512",
                },
                "logo": "http://i1.au.reastatic.net/agencylogo/XRWXMT/12/20120927204448.gif",
              },
              "id": "1",
              "mainImage": "http://i2.au.reastatic.net/640x480/20bfc8668a30e8cabf045a1cd54814a9042fc715a8be683ba196898333d68cec/main.jpg",
              "price": "$726,500",
            },
            Object {
              "agency": Object {
                "brandingColors": Object {
                  "primary": "#fcfa3b",
                },
                "logo": "http://i4.au.reastatic.net/agencylogo/BFERIC/12/20150619122858.gif",
              },
              "id": "2",
              "mainImage": "http://i1.au.reastatic.net/640x480/88586227f9176f602d5c19cf06261108dbb29f03e30d1c4ce9fc2b51fb1e4bd6/main.jpg",
              "price": "$560,520",
            },
            Object {
              "agency": Object {
                "brandingColors": Object {
                  "primary": "#57B5E0",
                },
                "logo": "http://i1.au.reastatic.net/agencylogo/XCEWIN/12/20150807093203.gif",
              },
              "id": "3",
              "mainImage": "http://i4.au.reastatic.net/640x480/98cee1b2a3a64329921fc38f7e2926a78d41fcc683fc48fb8a8ef2999b14c027/main.jpg",
              "price": "$826,500",
            },
          ]
    }
    saved={
        Array [
            Object {
              "agency": Object {
                "brandingColors": Object {
                  "primary": "#000000",
                },
                "logo": "http://i2.au.reastatic.net/agencylogo/WVYSSK/2/20140701084436.gif",
              },
              "id": "4",
              "mainImage": "http://i2.au.reastatic.net/640x480/5e84d96722dda3ea2a084d6935677f64872d1d760562d530c3cabfcb7bcda9c2/main.jpg",
              "price": "$526,500",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="propertyTypes"
          id="results"
>
          Results
          <div
                    className="propertyListStyles"
          >
                    <REProperty
                              cardProperty={
                                        Object {
                                                  "agency": Object {
                                                    "brandingColors": Object {
                                                      "primary": "#ffe512",
                                                    },
                                                    "logo": "http://i1.au.reastatic.net/agencylogo/XRWXMT/12/20120927204448.gif",
                                                  },
                                                  "id": "1",
                                                  "mainImage": "http://i2.au.reastatic.net/640x480/20bfc8668a30e8cabf045a1cd54814a9042fc715a8be683ba196898333d68cec/main.jpg",
                                                  "price": "$726,500",
                                                }
                              }
                    />
                    <REProperty
                              cardProperty={
                                        Object {
                                                  "agency": Object {
                                                    "brandingColors": Object {
                                                      "primary": "#fcfa3b",
                                                    },
                                                    "logo": "http://i4.au.reastatic.net/agencylogo/BFERIC/12/20150619122858.gif",
                                                  },
                                                  "id": "2",
                                                  "mainImage": "http://i1.au.reastatic.net/640x480/88586227f9176f602d5c19cf06261108dbb29f03e30d1c4ce9fc2b51fb1e4bd6/main.jpg",
                                                  "price": "$560,520",
                                                }
                              }
                    />
                    <REProperty
                              cardProperty={
                                        Object {
                                                  "agency": Object {
                                                    "brandingColors": Object {
                                                      "primary": "#57B5E0",
                                                    },
                                                    "logo": "http://i1.au.reastatic.net/agencylogo/XCEWIN/12/20150807093203.gif",
                                                  },
                                                  "id": "3",
                                                  "mainImage": "http://i4.au.reastatic.net/640x480/98cee1b2a3a64329921fc38f7e2926a78d41fcc683fc48fb8a8ef2999b14c027/main.jpg",
                                                  "price": "$826,500",
                                                }
                              }
                    />
          </div>
</div>,
        <div
          className="propertyTypes"
          id="saved"
>
          Saved
          <div
                    className="propertyListStyles"
          >
                    <REProperty
                              cardProperty={
                                        Object {
                                                  "agency": Object {
                                                    "brandingColors": Object {
                                                      "primary": "#000000",
                                                    },
                                                    "logo": "http://i2.au.reastatic.net/agencylogo/WVYSSK/2/20140701084436.gif",
                                                  },
                                                  "id": "4",
                                                  "mainImage": "http://i2.au.reastatic.net/640x480/5e84d96722dda3ea2a084d6935677f64872d1d760562d530c3cabfcb7bcda9c2/main.jpg",
                                                  "price": "$526,500",
                                                }
                              }
                    />
          </div>
</div>,
      ],
      "className": "propertyStyles",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Results",
            <div
              className="propertyListStyles"
>
              <REProperty
                            cardProperty={
                                          Object {
                                                        "agency": Object {
                                                          "brandingColors": Object {
                                                            "primary": "#ffe512",
                                                          },
                                                          "logo": "http://i1.au.reastatic.net/agencylogo/XRWXMT/12/20120927204448.gif",
                                                        },
                                                        "id": "1",
                                                        "mainImage": "http://i2.au.reastatic.net/640x480/20bfc8668a30e8cabf045a1cd54814a9042fc715a8be683ba196898333d68cec/main.jpg",
                                                        "price": "$726,500",
                                                      }
                            }
              />
              <REProperty
                            cardProperty={
                                          Object {
                                                        "agency": Object {
                                                          "brandingColors": Object {
                                                            "primary": "#fcfa3b",
                                                          },
                                                          "logo": "http://i4.au.reastatic.net/agencylogo/BFERIC/12/20150619122858.gif",
                                                        },
                                                        "id": "2",
                                                        "mainImage": "http://i1.au.reastatic.net/640x480/88586227f9176f602d5c19cf06261108dbb29f03e30d1c4ce9fc2b51fb1e4bd6/main.jpg",
                                                        "price": "$560,520",
                                                      }
                            }
              />
              <REProperty
                            cardProperty={
                                          Object {
                                                        "agency": Object {
                                                          "brandingColors": Object {
                                                            "primary": "#57B5E0",
                                                          },
                                                          "logo": "http://i1.au.reastatic.net/agencylogo/XCEWIN/12/20150807093203.gif",
                                                        },
                                                        "id": "3",
                                                        "mainImage": "http://i4.au.reastatic.net/640x480/98cee1b2a3a64329921fc38f7e2926a78d41fcc683fc48fb8a8ef2999b14c027/main.jpg",
                                                        "price": "$826,500",
                                                      }
                            }
              />
</div>,
          ],
          "className": "propertyTypes",
          "id": "results",
        },
        "ref": null,
        "rendered": Array [
          "Results",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <REProperty
                  cardProperty={
                                    Object {
                                                      "agency": Object {
                                                        "brandingColors": Object {
                                                          "primary": "#ffe512",
                                                        },
                                                        "logo": "http://i1.au.reastatic.net/agencylogo/XRWXMT/12/20120927204448.gif",
                                                      },
                                                      "id": "1",
                                                      "mainImage": "http://i2.au.reastatic.net/640x480/20bfc8668a30e8cabf045a1cd54814a9042fc715a8be683ba196898333d68cec/main.jpg",
                                                      "price": "$726,500",
                                                    }
                  }
/>,
                <REProperty
                  cardProperty={
                                    Object {
                                                      "agency": Object {
                                                        "brandingColors": Object {
                                                          "primary": "#fcfa3b",
                                                        },
                                                        "logo": "http://i4.au.reastatic.net/agencylogo/BFERIC/12/20150619122858.gif",
                                                      },
                                                      "id": "2",
                                                      "mainImage": "http://i1.au.reastatic.net/640x480/88586227f9176f602d5c19cf06261108dbb29f03e30d1c4ce9fc2b51fb1e4bd6/main.jpg",
                                                      "price": "$560,520",
                                                    }
                  }
/>,
                <REProperty
                  cardProperty={
                                    Object {
                                                      "agency": Object {
                                                        "brandingColors": Object {
                                                          "primary": "#57B5E0",
                                                        },
                                                        "logo": "http://i1.au.reastatic.net/agencylogo/XCEWIN/12/20150807093203.gif",
                                                      },
                                                      "id": "3",
                                                      "mainImage": "http://i4.au.reastatic.net/640x480/98cee1b2a3a64329921fc38f7e2926a78d41fcc683fc48fb8a8ef2999b14c027/main.jpg",
                                                      "price": "$826,500",
                                                    }
                  }
/>,
              ],
              "className": "propertyListStyles",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": "1",
                "nodeType": "function",
                "props": Object {
                  "cardProperty": Object {
                    "agency": Object {
                      "brandingColors": Object {
                        "primary": "#ffe512",
                      },
                      "logo": "http://i1.au.reastatic.net/agencylogo/XRWXMT/12/20120927204448.gif",
                    },
                    "id": "1",
                    "mainImage": "http://i2.au.reastatic.net/640x480/20bfc8668a30e8cabf045a1cd54814a9042fc715a8be683ba196898333d68cec/main.jpg",
                    "price": "$726,500",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "2",
                "nodeType": "function",
                "props": Object {
                  "cardProperty": Object {
                    "agency": Object {
                      "brandingColors": Object {
                        "primary": "#fcfa3b",
                      },
                      "logo": "http://i4.au.reastatic.net/agencylogo/BFERIC/12/20150619122858.gif",
                    },
                    "id": "2",
                    "mainImage": "http://i1.au.reastatic.net/640x480/88586227f9176f602d5c19cf06261108dbb29f03e30d1c4ce9fc2b51fb1e4bd6/main.jpg",
                    "price": "$560,520",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "3",
                "nodeType": "function",
                "props": Object {
                  "cardProperty": Object {
                    "agency": Object {
                      "brandingColors": Object {
                        "primary": "#57B5E0",
                      },
                      "logo": "http://i1.au.reastatic.net/agencylogo/XCEWIN/12/20150807093203.gif",
                    },
                    "id": "3",
                    "mainImage": "http://i4.au.reastatic.net/640x480/98cee1b2a3a64329921fc38f7e2926a78d41fcc683fc48fb8a8ef2999b14c027/main.jpg",
                    "price": "$826,500",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Saved",
            <div
              className="propertyListStyles"
>
              <REProperty
                            cardProperty={
                                          Object {
                                                        "agency": Object {
                                                          "brandingColors": Object {
                                                            "primary": "#000000",
                                                          },
                                                          "logo": "http://i2.au.reastatic.net/agencylogo/WVYSSK/2/20140701084436.gif",
                                                        },
                                                        "id": "4",
                                                        "mainImage": "http://i2.au.reastatic.net/640x480/5e84d96722dda3ea2a084d6935677f64872d1d760562d530c3cabfcb7bcda9c2/main.jpg",
                                                        "price": "$526,500",
                                                      }
                            }
              />
</div>,
          ],
          "className": "propertyTypes",
          "id": "saved",
        },
        "ref": null,
        "rendered": Array [
          "Saved",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <REProperty
                  cardProperty={
                                    Object {
                                                      "agency": Object {
                                                        "brandingColors": Object {
                                                          "primary": "#000000",
                                                        },
                                                        "logo": "http://i2.au.reastatic.net/agencylogo/WVYSSK/2/20140701084436.gif",
                                                      },
                                                      "id": "4",
                                                      "mainImage": "http://i2.au.reastatic.net/640x480/5e84d96722dda3ea2a084d6935677f64872d1d760562d530c3cabfcb7bcda9c2/main.jpg",
                                                      "price": "$526,500",
                                                    }
                  }
/>,
              ],
              "className": "propertyListStyles",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": "4",
                "nodeType": "function",
                "props": Object {
                  "cardProperty": Object {
                    "agency": Object {
                      "brandingColors": Object {
                        "primary": "#000000",
                      },
                      "logo": "http://i2.au.reastatic.net/agencylogo/WVYSSK/2/20140701084436.gif",
                    },
                    "id": "4",
                    "mainImage": "http://i2.au.reastatic.net/640x480/5e84d96722dda3ea2a084d6935677f64872d1d760562d530c3cabfcb7bcda9c2/main.jpg",
                    "price": "$526,500",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="propertyTypes"
            id="results"
>
            Results
            <div
                        className="propertyListStyles"
            >
                        <REProperty
                                    cardProperty={
                                                Object {
                                                            "agency": Object {
                                                              "brandingColors": Object {
                                                                "primary": "#ffe512",
                                                              },
                                                              "logo": "http://i1.au.reastatic.net/agencylogo/XRWXMT/12/20120927204448.gif",
                                                            },
                                                            "id": "1",
                                                            "mainImage": "http://i2.au.reastatic.net/640x480/20bfc8668a30e8cabf045a1cd54814a9042fc715a8be683ba196898333d68cec/main.jpg",
                                                            "price": "$726,500",
                                                          }
                                    }
                        />
                        <REProperty
                                    cardProperty={
                                                Object {
                                                            "agency": Object {
                                                              "brandingColors": Object {
                                                                "primary": "#fcfa3b",
                                                              },
                                                              "logo": "http://i4.au.reastatic.net/agencylogo/BFERIC/12/20150619122858.gif",
                                                            },
                                                            "id": "2",
                                                            "mainImage": "http://i1.au.reastatic.net/640x480/88586227f9176f602d5c19cf06261108dbb29f03e30d1c4ce9fc2b51fb1e4bd6/main.jpg",
                                                            "price": "$560,520",
                                                          }
                                    }
                        />
                        <REProperty
                                    cardProperty={
                                                Object {
                                                            "agency": Object {
                                                              "brandingColors": Object {
                                                                "primary": "#57B5E0",
                                                              },
                                                              "logo": "http://i1.au.reastatic.net/agencylogo/XCEWIN/12/20150807093203.gif",
                                                            },
                                                            "id": "3",
                                                            "mainImage": "http://i4.au.reastatic.net/640x480/98cee1b2a3a64329921fc38f7e2926a78d41fcc683fc48fb8a8ef2999b14c027/main.jpg",
                                                            "price": "$826,500",
                                                          }
                                    }
                        />
            </div>
</div>,
          <div
            className="propertyTypes"
            id="saved"
>
            Saved
            <div
                        className="propertyListStyles"
            >
                        <REProperty
                                    cardProperty={
                                                Object {
                                                            "agency": Object {
                                                              "brandingColors": Object {
                                                                "primary": "#000000",
                                                              },
                                                              "logo": "http://i2.au.reastatic.net/agencylogo/WVYSSK/2/20140701084436.gif",
                                                            },
                                                            "id": "4",
                                                            "mainImage": "http://i2.au.reastatic.net/640x480/5e84d96722dda3ea2a084d6935677f64872d1d760562d530c3cabfcb7bcda9c2/main.jpg",
                                                            "price": "$526,500",
                                                          }
                                    }
                        />
            </div>
</div>,
        ],
        "className": "propertyStyles",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Results",
              <div
                className="propertyListStyles"
>
                <REProperty
                                cardProperty={
                                                Object {
                                                                "agency": Object {
                                                                  "brandingColors": Object {
                                                                    "primary": "#ffe512",
                                                                  },
                                                                  "logo": "http://i1.au.reastatic.net/agencylogo/XRWXMT/12/20120927204448.gif",
                                                                },
                                                                "id": "1",
                                                                "mainImage": "http://i2.au.reastatic.net/640x480/20bfc8668a30e8cabf045a1cd54814a9042fc715a8be683ba196898333d68cec/main.jpg",
                                                                "price": "$726,500",
                                                              }
                                }
                />
                <REProperty
                                cardProperty={
                                                Object {
                                                                "agency": Object {
                                                                  "brandingColors": Object {
                                                                    "primary": "#fcfa3b",
                                                                  },
                                                                  "logo": "http://i4.au.reastatic.net/agencylogo/BFERIC/12/20150619122858.gif",
                                                                },
                                                                "id": "2",
                                                                "mainImage": "http://i1.au.reastatic.net/640x480/88586227f9176f602d5c19cf06261108dbb29f03e30d1c4ce9fc2b51fb1e4bd6/main.jpg",
                                                                "price": "$560,520",
                                                              }
                                }
                />
                <REProperty
                                cardProperty={
                                                Object {
                                                                "agency": Object {
                                                                  "brandingColors": Object {
                                                                    "primary": "#57B5E0",
                                                                  },
                                                                  "logo": "http://i1.au.reastatic.net/agencylogo/XCEWIN/12/20150807093203.gif",
                                                                },
                                                                "id": "3",
                                                                "mainImage": "http://i4.au.reastatic.net/640x480/98cee1b2a3a64329921fc38f7e2926a78d41fcc683fc48fb8a8ef2999b14c027/main.jpg",
                                                                "price": "$826,500",
                                                              }
                                }
                />
</div>,
            ],
            "className": "propertyTypes",
            "id": "results",
          },
          "ref": null,
          "rendered": Array [
            "Results",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <REProperty
                    cardProperty={
                                        Object {
                                                            "agency": Object {
                                                              "brandingColors": Object {
                                                                "primary": "#ffe512",
                                                              },
                                                              "logo": "http://i1.au.reastatic.net/agencylogo/XRWXMT/12/20120927204448.gif",
                                                            },
                                                            "id": "1",
                                                            "mainImage": "http://i2.au.reastatic.net/640x480/20bfc8668a30e8cabf045a1cd54814a9042fc715a8be683ba196898333d68cec/main.jpg",
                                                            "price": "$726,500",
                                                          }
                    }
/>,
                  <REProperty
                    cardProperty={
                                        Object {
                                                            "agency": Object {
                                                              "brandingColors": Object {
                                                                "primary": "#fcfa3b",
                                                              },
                                                              "logo": "http://i4.au.reastatic.net/agencylogo/BFERIC/12/20150619122858.gif",
                                                            },
                                                            "id": "2",
                                                            "mainImage": "http://i1.au.reastatic.net/640x480/88586227f9176f602d5c19cf06261108dbb29f03e30d1c4ce9fc2b51fb1e4bd6/main.jpg",
                                                            "price": "$560,520",
                                                          }
                    }
/>,
                  <REProperty
                    cardProperty={
                                        Object {
                                                            "agency": Object {
                                                              "brandingColors": Object {
                                                                "primary": "#57B5E0",
                                                              },
                                                              "logo": "http://i1.au.reastatic.net/agencylogo/XCEWIN/12/20150807093203.gif",
                                                            },
                                                            "id": "3",
                                                            "mainImage": "http://i4.au.reastatic.net/640x480/98cee1b2a3a64329921fc38f7e2926a78d41fcc683fc48fb8a8ef2999b14c027/main.jpg",
                                                            "price": "$826,500",
                                                          }
                    }
/>,
                ],
                "className": "propertyListStyles",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": "1",
                  "nodeType": "function",
                  "props": Object {
                    "cardProperty": Object {
                      "agency": Object {
                        "brandingColors": Object {
                          "primary": "#ffe512",
                        },
                        "logo": "http://i1.au.reastatic.net/agencylogo/XRWXMT/12/20120927204448.gif",
                      },
                      "id": "1",
                      "mainImage": "http://i2.au.reastatic.net/640x480/20bfc8668a30e8cabf045a1cd54814a9042fc715a8be683ba196898333d68cec/main.jpg",
                      "price": "$726,500",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": "2",
                  "nodeType": "function",
                  "props": Object {
                    "cardProperty": Object {
                      "agency": Object {
                        "brandingColors": Object {
                          "primary": "#fcfa3b",
                        },
                        "logo": "http://i4.au.reastatic.net/agencylogo/BFERIC/12/20150619122858.gif",
                      },
                      "id": "2",
                      "mainImage": "http://i1.au.reastatic.net/640x480/88586227f9176f602d5c19cf06261108dbb29f03e30d1c4ce9fc2b51fb1e4bd6/main.jpg",
                      "price": "$560,520",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": "3",
                  "nodeType": "function",
                  "props": Object {
                    "cardProperty": Object {
                      "agency": Object {
                        "brandingColors": Object {
                          "primary": "#57B5E0",
                        },
                        "logo": "http://i1.au.reastatic.net/agencylogo/XCEWIN/12/20150807093203.gif",
                      },
                      "id": "3",
                      "mainImage": "http://i4.au.reastatic.net/640x480/98cee1b2a3a64329921fc38f7e2926a78d41fcc683fc48fb8a8ef2999b14c027/main.jpg",
                      "price": "$826,500",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Saved",
              <div
                className="propertyListStyles"
>
                <REProperty
                                cardProperty={
                                                Object {
                                                                "agency": Object {
                                                                  "brandingColors": Object {
                                                                    "primary": "#000000",
                                                                  },
                                                                  "logo": "http://i2.au.reastatic.net/agencylogo/WVYSSK/2/20140701084436.gif",
                                                                },
                                                                "id": "4",
                                                                "mainImage": "http://i2.au.reastatic.net/640x480/5e84d96722dda3ea2a084d6935677f64872d1d760562d530c3cabfcb7bcda9c2/main.jpg",
                                                                "price": "$526,500",
                                                              }
                                }
                />
</div>,
            ],
            "className": "propertyTypes",
            "id": "saved",
          },
          "ref": null,
          "rendered": Array [
            "Saved",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <REProperty
                    cardProperty={
                                        Object {
                                                            "agency": Object {
                                                              "brandingColors": Object {
                                                                "primary": "#000000",
                                                              },
                                                              "logo": "http://i2.au.reastatic.net/agencylogo/WVYSSK/2/20140701084436.gif",
                                                            },
                                                            "id": "4",
                                                            "mainImage": "http://i2.au.reastatic.net/640x480/5e84d96722dda3ea2a084d6935677f64872d1d760562d530c3cabfcb7bcda9c2/main.jpg",
                                                            "price": "$526,500",
                                                          }
                    }
/>,
                ],
                "className": "propertyListStyles",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": "4",
                  "nodeType": "function",
                  "props": Object {
                    "cardProperty": Object {
                      "agency": Object {
                        "brandingColors": Object {
                          "primary": "#000000",
                        },
                        "logo": "http://i2.au.reastatic.net/agencylogo/WVYSSK/2/20140701084436.gif",
                      },
                      "id": "4",
                      "mainImage": "http://i2.au.reastatic.net/640x480/5e84d96722dda3ea2a084d6935677f64872d1d760562d530c3cabfcb7bcda9c2/main.jpg",
                      "price": "$526,500",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
